webrtc.html

```diff
@@ -2404,24 +2404,24 @@ <h4>
                                   each of the simulcast layers, populating the
                                   {{RTCRtpCodingParameters/rid}} member
                                   according to the corresponding rid value, and
-                                  let <var>sendEncodings</var> be the list
+                                  let <var>receiveEncodings</var> be the list
```

Comment:
On careful review, it seems to be what's being proposed by the remote end that is the basis for modifying sender.sendEncoding. I suggest we call it proposedSendEncodings instead, to avoid confusion with a possible later extension for receive encodings.

(this was intended to be part of the first review comment, but somehow never landed in this comment)

Reply:
Yeah, I agree with the naming change to proposedSendEncodings.